src/Microsoft.AspNetCore.Mvc.Formatters.Xml/XmlDataContractSerializerOutputFormatter.cs

```diff
@@ -37,6 +49,16 @@ public class XmlDataContractSerializerOutputFormatter : TextOutputFormatter
         /// </summary>
         /// <param name="writerSettings">The settings to be used by the <see cref="DataContractSerializer"/>.</param>
         public XmlDataContractSerializerOutputFormatter(XmlWriterSettings writerSettings)
+            : this(writerSettings, logger: null)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="XmlDataContractSerializerOutputFormatter"/>
+        /// </summary>
+        /// <param name="writerSettings">The settings to be used by the <see cref="DataContractSerializer"/>.</param>
+        /// <param name="logger">The <see cref="ILogger"/>.</param>
+        public XmlDataContractSerializerOutputFormatter(XmlWriterSettings writerSettings, ILogger logger)
```

Comment:
Other than tests, where are the constructor overloads accepting a `XmlWriterSettings` expected to be used? I'm wondering if we can limit the additions to one overload per formatter type.